### ISSUE TITLE
Confusion about labeling the data

### ISSUE THREAD

**Issue by Mstfakts** [boschresearch/CNC_Machining#3]
Hello,

Before asking my question, I would like to thank for sharing this beautiful data with people.

I have confusued about how the data was labeled. There are many files under M1/OP1/good for example. Are these files sequences or are they independent trials? Another point not clear is the "bad" labels. Are they again a sequence that comes just after good lables or whole a bad file is a trial by itself?

To sum up my question, 
1- Are the files under M1/OP1 path is only one time-series trial that was randomly divided into sub-files?
2- What is the relation of "bad" labeled files with good labels? Whole one bad file considered as "bad" or only some part of it is bad?

Can you please clarify me about lableing the data?

**Comment by MATnani** [comment]
Thank you for your comment and interest in our dataset. As described in the paper, the data has been collected from real production, meaning that it has not been collected from trials. The production process contains a series of OPs (for example: OP5-OP1-OP4-...). During the labeling job, the production process has been segmented and labeled as OK/NOK and saved in a single file.  
To answer your questions:
1. No, the files does not belong to a single trial. Each file represents a full OP process which run in the machine during production and has been extracted and labeled. 
2. The "bad" label has been given by the expert on the shopfloor as a NOK process where an anomaly in the OP process has been detected (such as misalignment of the tool workpiece).
